src/overlays/gamestates/ovl_opening/z_opening.h

```diff
@@ -1,15 +1,15 @@
-#ifndef _Z64_OPENING_H_
-#define _Z64_OPENING_H_
+#ifndef Z64_OPENING_H
+#define Z64_OPENING_H
 
-#include "global.h"
-
-void TitleSetup_Init(GameState* thisx);
-void TitleSetup_Destroy(GameState* thisx);
+#include "z64game.h"
+#include "z64view.h"
 
 typedef struct {
     /* 0x000 */ GameState state;
-    /* 0x0A4 */ UNK_TYPE1 unk_A4[0x4];
     /* 0x0A8 */ View view;
```

Comment:
Does `View` still align to `0xA8`?

Reply:
Yes, because `View` has the `Vp vp;` member, which has 0x8 alignment, and that propagates over to `View`.
Because of that, `TitleSetupState` has 0x8 alignment too, but since this struct isn't a substruct of anything else, then it doesn't matter